app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PostsController < ApplicationController
+
+  def index
+    @posts = current_user.timeline_posts(current_user).flatten
```

Comment:
This makes us only see `current_user` 's posts. But what if i wanna see other users' posts also?

Reply:
Like I commented below with the attached screenshot this controller gets the current_user's post and the post of his friends. According to **Odin project, bullet point 13** that is what we are supposed to do, see the attached screenshot below to verify this